typewriter/fixes/base.py

```diff
@@ -741,5 +745,13 @@ def type_updater(self, match):
         else:
             mod, name = word.rsplit('.', 1)
             to_import = name
-        self.add_import(mod, to_import)
+
+        # Get the typename we need to use to be valid for a current import statement
+        result = type_by_import_stmt(mod, to_import, node)
+
+        if result is not None:
+            # There exists a current import statement this type is valid for
+            return result
+
+        self.add_import(mod, to_import)  # There exists no import statement for this type, add one
```

Comment:
add comment on its own line before this, rather than trailing after it.

Reply:
done